src/documentation/system_administrators/advanced/quantum_safe_storage/quantum_safe_storage_deploy.md

```diff
@@ -0,0 +1,594 @@
+
+```sh
+openssl rand -hex 32
+```
+
+Once this is done, take the output and insert it into the file in the indicated locations.
+
+### Prepare for Zdb Deployment
+
+Now we will deploy the zdbs and write their details into the config file in the proper format. Before proceeding, we'll set a few shell variables that will be used in the deployment scripts:
+
+```sh
+CONFIG=zstor-default.toml
+PASSWORD=$(openssl rand -base64 18)
+METADATA_NODES="1 2 3 4"
+back end_NODES="1 2 3 4 5 6 7 8"
+back end_SIZE=1
+```
+
+This will generate a strong random password that will be used to secure each zdb. You can replace the code that generates the password with your own password if you wish. For now, don't worry about having to save the randomly generated password. It will get written to the config file and you can take note of it later.
+
+For the metadata and back end nodes, replace the example values with the node IDs you selected before. Set your desired back end size too, which is specified in gigabtyes.
+
+
+### Deploy Metadata Zdbs
+
+Here is an example script to deploy the metadata zdbs based on the variables set above. This uses a fixed size of 1 GB, which should be plenty:
+
+```sh
+for node in $METADATA_NODES; do
+  name=node${node}meta
+  tfcmd deploy zdb --mode user --node $node -n $name --size 1 --password $PASSWORD
+done
+```
+
+It's possible that some of your chosen nodes don't respond properly at deployment time and need to be replaced with other nodes. In that case, just replace the variable with the new node IDs like this:
+
+```sh
+# Example: Node 3 wasn't working, replace it with node 5
+METADATA_NODES="1 2 4 5"
+```
+
+Then you can just run the original deployment script loop again. Any zdb deployments that would be duplicated will be detected by `tfcmd` and skipped over.
+
+### Cancel Metadata Zdbs
+
+If you need to cancel the metadata zdb deployments, use this script:
+
+```sh
+for node in $METADATA_NODES; do
+  name=node${node}meta
+  tfcmd cancel $name
+done
+```
+
+### Write Metadata Zdbs Config
+
+Next we will write the configuration data from the deployed zdbs into the config file. The script as shown is for IPv6 connections. If you are using Yggdrasil, replace `.Zdbs[0].ips[0]` with `.Zdbs[0].ips[1]`
+
+```sh
+# Wait about five seconds before doing the next step to make sure data is available. Only needed if running immediately after the deployment step in a single script
+sleep 5
+
+# Write out the config sections for the metadata zdbs
+for node in $METADATA_NODES; do
+  name=node${node}meta
+  echo Fetching and writing config for $name
+  json=$(tfcmd get zdb $name 2>&1 | tail -n +3 | sed $'s/\e\[0m//')
+  ip=$(echo $json | jq .Zdbs[0].ips[-1] | tr -d \")
+  port=$(echo $json | jq .Zdbs[0].port)
+  namespace=$(echo $json | jq .Zdbs[0].namespace)
+  password=$(echo $json | jq .Zdbs[0].password)
+
+  echo \# Node $node >> $CONFIG
+  echo [[meta.config.back ends]] >> $CONFIG
```

Comment:
It should be `[[meta.config.backends]]` with no space.

Reply:
This was from the same find/replace error. Is fixed.